doc/build/html/_sources/index.rst.txt

```diff
@@ -4,23 +4,51 @@
    contain the root `toctree` directive.
 
 Welcome to The Sheriff's documentation!
-=======================================
+***************************************
```

Comment:
I'd say this should be consider the Document title and should be:

```
================
 Document Title
================
```

Reply:
See my answer in Slack